src/utils/ErrorUtils.js

```diff
@@ -0,0 +1,60 @@
+//@flow
+import ErrorCodes from './ErrorCodes';
+import TCATUtils from './TCATUtils';
+
+function getErrorCode(message) {
+
+    message = message.toLowerCase();
+
+    if (message.includes('is out of bounds')) {
```

Comment:
can we move these to `const`s at the top of the file? it's not good practice to have hardcoded strings in the middle of the file 

```
const OUT_OF_BOUNDS_MESSAGE  = 'is out of bounds';
...
if (message.includes(OUT_OF_BOUNDS_MESSAGE))
```